packages/prop-controllers/src/responsive-opacity/responsive-opacity.ts

```diff
@@ -10,11 +10,12 @@ export type ResponsiveOpacityValue = z.infer<
   typeof responsiveOpacityValueSchema
 >
 
-export const ResponsiveOpacity = versionedPropDef<{}>()(
+export const ResponsiveOpacity = versionedPropDef(
   Types.ResponsiveOpacity,
   responsiveOpacityValueSchema,
   {
     version: 1,
     dataKey: 'prop-controllers::responsive-opacity::v1',
   },
+  typeArg<{}>(),
```

Comment:
The `{}` type is effectively `any` in TypeScript. For an empty options object, should we be using `Record<string, never>`?

See https://www.totaltypescript.com/the-empty-object-type-in-typescript.

Reply:
Good catch, it was actually `Record<string, never>` in the original code (https://github.com/makeswift/makeswift/pull/754/files#diff-1ecb7c462ca01ab9ac5cef93354d548cc023a085d175fcb3697ade52c2e831ffL183), but got mangled during transition to `versionedPropDef`.